### PR TITLE
Added Logic/Math Operators for Objective-C

### LINKED ISSUE #305: Add Objective-C Logic/Math Operators
# Description

Logic/Math Operators for Objective-C aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right Objective-C directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of Objective-C
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by riyakushwaha**
## What GitHub issue does this PR apply to?

Resolves #305

## What changed and why?
Added operators.json in web/thesauruses/objective-c folder 

## (If editing Django app) Please add screenshots
No changes

## Checklist

- [ ] I claimed any associated issue(s) and they are not someone else's
- [ ] I have looked at documentation to ensure I made any revisions correctly
- [ ] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/objectivec/operators.json [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    "decrement": {
+      "code": "int a = 10; a--;",
+      "comment": "Value of a is 9"
+    },
+    "exponential": {
+      "code": "^",
+      "comment": "Value of 5^3 is 125"
+    },
+    "factorial": {
+      "code": "",
```
Comment (geekygirlsarah) [inline web/thesauruses/objectivec/operators.json]: If a concept is `not-implemented`, you can remove the `code` line. (And for all of the ones below)

File: web/thesauruses/objectivec/operators.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+      "code":"",
+      "not-implemented": "true"
+    },
+    "xnor_assignment": {
+      "code": "",
+      "not-implemented": "true"
+    },
+    "left_shift": {
+      "code": "int a = 1 << 2",
+      "comment": "Shits the specified bits to left, so a equals to (4)100"
```
Comment (geekygirlsarah) [inline web/thesauruses/objectivec/operators.json]: Uh... I think this is a typo 😂

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I think this is close! Make the couple of changes and it should be good to merge!

**Comment by riyakushwaha** [comment]
Thanks for reviewing. I have made all the required changes and made a pull request. Kindly have a look.

File: web/thesauruses/objectivec/operators.json [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    },
+    "xnor": {
+      "not-implemented": "true"
+    },
+    "xnor_assignment": {
+      "not-implemented": "true"
+    },
+    "left_shift": {
+      "code": "int a = 1 << 2",
+      "comment": "Shits the specified bits to left, so a equals to 4"
```
Comment (geekygirlsarah) [inline web/thesauruses/objectivec/operators.json]: You still have a typo with "shift" here

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]

File: web/thesauruses/objectivec/operators.json (hunk 3 above)
Comment (riyakushwaha) [reply web/thesauruses/objectivec/operators.json]: Soory I missed this again. I have corrected it now.

**Comment by geekygirlsarah** [comment]
Thanks for that last change! I'll merge it in.

Don't forget to sign up for Hacktoberfest if you haven't and if you want the shirt/stickers!

**Pull request merged**